clouddq/classes/dq_rule_binding.py

```diff
@@ -92,13 +92,14 @@ def from_dict(
         )
         if column_id:
             column_id.upper()
-        row_filter_id: str = get_from_dict_and_assert(
+        row_filter_ids: list[str] = get_from_dict_and_assert(
             config_id=rule_binding_id,
             kwargs=kwargs,
-            key="row_filter_id",
+            key="row_filter_ids",
+            assertion=lambda x: type(x) == list,
+            error_msg=f"Rule Binding ID: '{rule_binding_id}' must have defined value "
+            f"'row_filter_ids' of type 'list'.",
         )
-        if row_filter_id:
-            row_filter_id.upper()
```

Comment:
I think we'll need to caps all the row_filter_ids as well

Reply:
it was in `resolve_row_filter_configs` function like rule-id but now, you can see it L.102-109